force-app/main/default/lwc/propertyFilter/__tests__/propertyFilter.test.js

```diff
@@ -231,7 +205,67 @@ describe('c-property-filter', () => {
             expect(publish).toHaveBeenCalledWith(
                 undefined,
                 FILTERSCHANGEMC,
-                SEARCH_CRITERIA
+                DEFAULT_SEARCH_CRITERIA
+            );
+        });
+    });
+
+    it('resets to default values when reset button is clicked', () => {
+        // Create initial element
+        const element = createElement('c-property-filter', {
+            is: PropertyFilter
+        });
+        document.body.appendChild(element);
```

Comment:
Set initial values to the input and sliders after adding the component to the DOM so that you can indeed validate that the values did change when you hit reset.

Do something like this:
```js
element.shadowRoot.querySelector('lightning-input').value = UPDATED_SEARCH_KEY;
...
```